app/src/test/java/ar/scacchipa/twittercloneapp/repository/AuthorizationRepositoryTest.kt

```diff
@@ -0,0 +1,97 @@
+package ar.scacchipa.twittercloneapp.repository
+
+import ar.scacchipa.twittercloneapp.datasource.IAuthDataSource
+import ar.scacchipa.twittercloneapp.datasource.UserAccessToken
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import okhttp3.MediaType
+import okhttp3.ResponseBody
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import retrofit2.Response
+
+@ExperimentalCoroutinesApi
+class AuthorizationRepositoryTest {
+
+    private var authorizationRepository: IAuthorizationRepository? = null
+
+    @Before
+    fun setup() {
+        authorizationRepository = AuthorizationRepository(MockAuthDataSource())
+    }
+
+    @Test
+    fun shouldReturnACorrectUserAccessToken() = runTest {
```

Comment:
maybe add the why before the should on the name of the test would make it eassier to understand what is actually testing

Reply:
New the name of this function became "repoShouldReturnASuccessUserAccessToken()"
and the name of the next function became "repoShouldReturnADeniedUserAccessToken()"